core/drivers/crypto/caam/caam_desc.c

```diff
@@ -80,7 +84,8 @@ void caam_desc_push(struct caam_inring_entry *in_entry, paddr_t paddr)
 
 paddr_t caam_desc_pop(struct caam_outring_entry *out_entry)
 {
-	const uint32_t *a32 = (const uint32_t *)(&out_entry->desc);
+	const uintptr_t v_desc = (uintptr_t)(&out_entry->desc);
```

Comment:
The parenthesis around `(&out_entry->desc)` are reduntant.

Reply:
Ok